routes/frontend.php

```diff
@@ -18,3 +18,12 @@
         'uses' => 'ResetPasswordController@done',
     ]);
 });
+
+Route::group(['namespace' => 'Nodes\Api\Auth\EmailVerification', 'prefix' => 'email-verification'], function() {
+
+    // Confirm email
+    Route::get('/{token}/{email}', [
+        'as' => 'nodes.api.auth.email-verificatio.confirm',
+        'uses' => 'EmailVerificationController@index',
```

Comment:
This is maybe a bit nazi, but we usually also name Controllers in plural. So in this case it would be EmailVerification**s**